include/json/reader.h

```diff
@@ -6,6 +6,11 @@
 #ifndef JSON_READER_H_INCLUDED
 #define JSON_READER_H_INCLUDED
 
+#if defined(_MSC_VER)
+// Disable warning about deprecated destructors
```

Comment:
What deprecated destructors? Should we just be updating the destructors instead of suppressing here?